server/registry/filtering/models.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filtering
+
+import (
+	"github.com/apigee/registry/rpc"
+	"github.com/apigee/registry/server/registry/internal/storage/models"
```

Comment:
It seems weird to move this package out of `internal` and have it depend on `internal/storage/models`.